File: go/internal/headers/headers.go
// Package headers defines the X-Cron-* HTTP header names cronix uses on
// signed manifest fetches and trigger requests. Keep in sync with
// ts/packages/sdk/src/core/headers.ts and DECISIONS.md D-016 / D-021.
package headers

const (
	Signature           = "X-Cron-Signature"
	RunID               = "X-Cron-Run-Id"
	ScheduleName        = "X-Cron-Schedule-Name"
	FireTime            = "X-Cron-Fire-Time"
	FireTimeActual      = "X-Cron-Fire-Time-Actual"
	Attempt             = "X-Cron-Attempt"
	PreviousSuccessTime = "X-Cron-Previous-Success-Time"
)

File: ts/packages/sdk/src/core/headers.ts
/**
 * X-Cron-* HTTP header names. These are the wire-format header constants
 * for trigger requests and signed manifest fetches. Keep in sync with
 * go/internal/headers/headers.go and DECISIONS.md D-016 / D-021.
 */
export const HeaderSignature = "X-Cron-Signature";
export const HeaderRunId = "X-Cron-Run-Id";
export const HeaderScheduleName = "X-Cron-Schedule-Name";
export const HeaderFireTime = "X-Cron-Fire-Time";
export const HeaderFireTimeActual = "X-Cron-Fire-Time-Actual";
export const HeaderAttempt = "X-Cron-Attempt";
export const HeaderPreviousSuccessTime = "X-Cron-Previous-Success-Time";
